src/components/Header.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable jsx-a11y/alt-text */
+import React, { useContext } from 'react'
+import {ProductContxt} from '../App'
+import '../style/Header.css' 
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCartShopping } from "@fortawesome/free-solid-svg-icons"
+
+function IamgeCombonant() {
+  return (
+    <>
+      <div className='image'><div>Men</div></div>
+    </>
+  )
+}
+function Header() {
+  const {clicked, setClicked} = useContext(ProductContxt);
+  return (
+    <>
+    <div className='header'>
+      <div className='logo'>
+        <img
+        src='https://cdn.shopify.com/s/files/1/0665/2889/files/Thesus_A_W_Wordmark_Forest_Green_2cca776c-1727-4416-868a-fa72e7359f08.png?v=1635783797'
+            style={{ width: '200px' }}
+        />
+        </div>
+      <div className='links' >
+        <p className='Header_link '>Weekend Boots</p>
```

Comment:
Create an array of strings at the start of the page before rendring:
`const headers=["Weekend Boots","Accessories",etc..]
`
then map it in the rendering

`headers.map(header=> <p className='Header_link '>{header}</p>)
`

Reply:
done